src/patternfly/components/Table/TableCells/table-cell.hbs

```diff
@@ -0,0 +1,11 @@
+{{!-- TODO: update `table-td` to `table-cell` --}}
+{{!-- TODO: automate `td/th` based up `IsThead/IsTbody` and make overridable --}}
+{{!-- TODO: automate attribution --}}
+
+{{#> config table-cell--tag=(ternary IsThead 'th' 'td')}}
+
+<{{table-cell--tag}} class="{{concat (pfv) 'table__' (ternary isThead 'th' 'td') ' ' (concat (pfv) 'table' table-cell--type)}}" {{#if table-cell--attribute}}{{{table-cell--attribute}}}{{/if}}>
+  {{> @partial-block}}
+</{{table-cell--tag}}>
```

Comment:
We also need to be able to add an aria-label to the `th` element, in instances where there is visible content but no visible text content (checkbox, plain icon button, etc). The checkbox example in React has `<th aria-label="Row select" ....`, while the checkbox inside the th has its own aria-label to denote the action.

Core examples that would need an aria-label on the outer `th` element:

- Checkboxes and actions example
- Expandable example (both the "toggle all" button and "select all" checkbox)
- Expandable with set width columns example
- Expandable with nested table example (from what I can tell, the aria-labelledby on the "toggle all rows" button could be removed as it's not pointing to anything valid/doesn't need to. The aria-labelledby on the individual row toggles could be removed and the aria-label updated to e.g. "Toggle row 1", or the aria-labelledby needs to point to a valid element as they currently do not. This isn't a blocker for this PR, though)
- Compact example
- Compact expandable example
- Clickable example
- Clickable and expandable example
- Borderless example
- Borderless compact example
- Borderless expandable example
- Width modifiers examples
- Nested column headers and expandable rows
- Favorites examples
- Favorites sortable example (would also apply to your other PR for favorite sortable)
- Striped expandable table example